android/src/main/java/com/bugsnag/BugsnagReactNative.java

```diff
@@ -107,13 +107,9 @@ public void leaveBreadcrumb(ReadableMap options) {
                               readStringMap(options.getMap("metadata")));
   }
 
-  @ReactMethod
-  public void notify(ReadableMap payload) {
-      notifyBlocking(payload, false, null);
-  }
 
   @ReactMethod
-  public void notifyBlocking(ReadableMap payload, boolean blocking, com.facebook.react.bridge.Callback callback) {
+  public void notify(ReadableMap payload, com.facebook.react.bridge.Promise promise) {
```

Comment:
We could get away without the fully qualified class name here - it was necessary previously to disambiguate between the two `Callback` classes